pkg/machines/machines.less

```diff
@@ -6,9 +6,6 @@
 
 // For patternfly-react and @patternfly/react-console
 @import "~xterm/lib/xterm.css";
```

Comment:
Seems you wanted to remove this in a separate commit instead, but there is none?

Reply:
Still outstanding?

Reply:
Ah sorry forgot to answer. I don't think this should be there, but I don't see why this removal should be part of this PR either. Does not get anyhow affected by the PF4 things.